app/src/components/users.js

```diff
@@ -1,9 +1,31 @@
+const config = require('config');
 const log = require('npmlog');
 
+const KeyCloakServiceClientManager = require('./keyCloakServiceClientMgr');
+const RealmAdminService = require('./realmAdminSvc');
 const { userService } = require('../services');
 const utils = require('./utils');
 
 const users = {
+  /**
+  * @function getAllGetokUsers
+  * Returns all the users in the keycloak realm that the GETOK api is running on. Use the api's client to call realm api
+  * @returns {object[]} An array of user objects
+  */
+  getAllGetokUsers: async () => {
+    log.debug('getAllGetokUsers');
```

Comment:
Half tempted to say this should be on verbose level as it's only letting you know you entered the function call.

Reply:
Yeah it's a trace statement so changing to verbose